crates/iota-json-rpc-api/src/governance.rs

```diff
@@ -53,15 +53,17 @@ pub trait GovernanceReadApi {
         epoch: Option<BigInt<u64>>,
     ) -> RpcResult<IotaCommittee>;
 
-    /// Return the latest IOTA system state object on-chain.
-    #[method(name = "getLatestIotaSystemState")]
-    async fn get_latest_iota_system_state(&self) -> RpcResult<IotaSystemStateSummaryV2>;
+    /// Return the latest IOTA system state object on networks supporting
+    /// protocol version `>= 5`. These are networks with node software release
+    /// version `>= 0.11`.
+    #[method(name = "getLatestIotaSystemStateV2")]
+    async fn get_latest_iota_system_state_v2(&self) -> RpcResult<IotaSystemStateSummary>;
 
-    /// Return the latest IOTA system state object on-chain (version 1).
-    /// Requires the `client-target-api-version` header to be set into
-    /// a value `< 0.11` during requests.
-    #[method(name = "getLatestIotaSystemState", version <= "0.10.99")]
-    async fn get_latest_iota_system_state_v1(&self) -> RpcResult<IotaSystemStateSummaryV1>;
+    /// Return the latest IOTA system state object on networks supporting
+    /// protocol version `< 4`. These are networks with node software release
+    /// version `< 0.11`.
+    #[method(name = "getLatestIotaSystemState")]
+    async fn get_latest_iota_system_state(&self) -> RpcResult<IotaSystemStateSummaryV1>;
```

Comment:
I'm wondering, should we add a deprecated attribute here so it's more obvious for devs that they probably want v2?

Reply:
Sounds like the second option makes more sense then

Reply:
https://github.com/iotaledger/iota/pull/5586/commits/fef43c90547f0a770c47d400cd00c1aff5fe5dc0